### PR TITLE
style: remove repeate method

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [x] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | style: remove repeate method |
| 🇨🇳 Chinese | style: remove repeate method |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/button/button-group.tsx [hunk 1]
```diff
@@ -35,6 +35,7 @@ const ButtonGroup: React.FC<ButtonGroupProps> = props => {
       break;
     default:
       warning(!size, 'Button.Group', 'Invalid prop `size`.');
+      break;
```
Comment (afc163) [inline components/button/button-group.tsx:38]: 为啥要加这个。

Comment (li-jia-nan) [reply components/button/button-group.tsx:38]: 这里是最后一个 case，不加 break 也没啥问题

我个人觉得这样严谨一点，正好看到就顺手加上了

Comment (afc163) [reply components/button/button-group.tsx:38]: 没问题就不动吧，default 没必要加 break。

Comment (li-jia-nan) [reply components/button/button-group.tsx]: OK，去掉了

**Review by afc163 — APPROVED** [review]

**Pull request merged**